### ISSUE TITLE
Wrong Proptype in the (yarn) published version

### ISSUE THREAD

**Issue by yaseenshaik** [ealush/emoji-picker#37]
The proptype for `preload` should be `.bool` as in the current github version instead of `boolean` in the published version.

```js
EmojiPicker.propTypes = {
    ...
    preload: PropTypes.boolean
};
```

**Comment by ealush** [comment]
Hey @yaseenshaik, thanks for reporting this. 👍 
Can you please make sure that you are using the latest version of the picker? The issue you are describing was introduced in `1.5.3` and fixed in `1.5.4`. Latest version seems to operate normally on my end.

**Comment by ealush** [comment]
No other complaints, and it seems to be working fine on my end. Closing.
